tests/adiabatic_initial_conditions_chunk/screen-output

```diff
@@ -5,31 +5,31 @@ Number of degrees of freedom: 1,444 (882+121+441)
 *** Timestep 0:  t=0 seconds
    Solving temperature system... 0 iterations.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 26+0 iterations.
+   Solving Stokes system... 25+0 iterations.
```

Comment:
Oh, I had looked at `adiabatic_initial_conditions` by accident. Thanks for reminding me.

It is a bit of a shame that this patch increases the number of iterations (in further time steps) by so much :-( But it is strictly more correct, so I think we should still go with it.